src/core/reference/include/ngraph/runtime/reference/fake_quantize.hpp

```diff
@@ -105,9 +105,7 @@ class QuantizationBound {
             bound = Bound::aligned;
         } else {
             bound = Bound::broadcast;
-            const auto arg_memory_offsets = row_major_strides(arg_shape);
-            const auto unsqueezed_bound_shape = align_shape_sizes(bound_shape, arg_shape, broadcast_spec);
-            row_strides = calc_broadcast_index_offset(arg_memory_offsets, unsqueezed_bound_shape);
+            row_strides = row_major_strides(arg_shape);
```

Comment:
Looks like methods `align_shape_sizes` and `calc_broadcast_index_offset` were used only from this place. I think we can remove them.

Reply:
Done